Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -705,6 +705,20 @@ void PreBattleCalcSkills(struct BattleUnit * attacker, struct BattleUnit * defen
         attacker->battleHitRate += 10;
     }
 #endif
+
+#if (defined(SID_SilentPride) && (SID_SilentPride < MAX_SKILL_NUM))
+    if (SkillTester(unit, SID_SilentPride))
+    {
+        int lostHP = GetUnitMaxHp(unit) - GetUnitCurrentHp(unit);
+
+        if (lostHP > 3)
+        {
+            int multiplier = lostHP % 4;
```

Comment:
why "%"？

Reply:
Because I need to get how many quarters (25%) that the lost HP is. If a unit has 16 base HP and loses 9, then the multiplier will be 2.